### PR TITLE
Updated Strings

### PR DESCRIPTION
**Pull request by zekroman**
Hopefully everything checks out :0

### CODE REVIEW

**Comment by meemo** [comment]
Looks good, just need to create the ignores file for archive.po (ignores/pt_PT/archive.txt):

FLIPNOTE_ARCHIVE
FLIPNOTES
MONTH_YEAR_FORMAT
NAVBAR_FAQS
REGION_AMERICAS
SPINOFFS
SPOTIFY
SUDOMEMO
YOUTUBE_MUSIC

File: pt_PT/LC_MESSAGES/creatorsRoom.po [hunk 1]
```diff
@@ -154,7 +154,7 @@ msgid "VIEW"
 msgstr "Ver"
 
 msgid "FAN_COUNT_FORMAT"
-msgstr "Fans: %s"
+msgstr "Fãs: %s"
```
Comment (meemo) [inline pt_PT/LC_MESSAGES/creatorsRoom.po:157]: I'm not sure if this character will appear at all on the DS, but we can still merge and correct it in a subsequent PR if it doesn't appear.

Comment (Tescu48) [reply pt_PT/LC_MESSAGES/creatorsRoom.po:157]: Oh damn idk wtf I did there. Should be "Fani"

Comment (zekroman) [reply pt_PT/LC_MESSAGES/creatorsRoom.po:157]: The PicToChat keyboard does have that character, so it's safe to say that said character can be added in

**Comment by meemo** [comment]
Looks good, thanks!

**Pull request merged**